### DISCUSSION TITLE
能否增加关闭右键动画开关

### DISCUSSION THREAD

**Discussion by LFRon** [linuxdeepin/developer-center#10985] [Features Request & Ideas | 特性请求 & 头脑风暴]
我的赛扬3865U处理器的笔记本跑deepin,右键菜单非常不跟手,后来我试了在AMD Ryzen 7 8845HS的Radeon 780M核显上跑仍然很不跟手,这个动画严重影响了生产力和工作效率,能否在设置中增加开关允许将其关闭?谢谢!

**Comment by BLumia** [comment]
~~好像是已经加了，但还没推送？~~ cc @kegechen 

确认已经改了，更新到 dtk 5.7.3 版本就会有，并且默认关闭。当前内测应该已经是这个版本了。
